source/extensions/filters/network/dubbo_proxy/conn_manager.cc

```diff
@@ -68,6 +68,12 @@ void ConnectionManager::onEvent(Network::ConnectionEvent event) {
   resetAllMessages(event == Network::ConnectionEvent::LocalClose);
 }
 
+void ConnectionManager::onAboveWriteBufferOverflowWatermark() {
+  // TODO(adip): Test counters
```

Comment:
For some extensions, I think these are no-ops and don't need any stats or treatment, I guess where there are explicit buffers things are different. Do you know which extensions might be doing interesting behaviors around buffer management and would need to action on overflow watermark?

Reply:
I agree, especially as the stats add much more noise to the admins.
Generally I've only added stats to places where High/Low watermarks were actually used, which might indicate that the buffer should be limited in size.
In this specific case the High and Low watermarks apply the `readDisable()` method on the connection, and so I've added the stats counter.
In places where there's a consensus that it should be omitted, I'll remove them.